api/types/role.go

```diff
@@ -306,7 +306,7 @@ func NewRole(name string, spec RoleSpecV6) (Role, error) {
 	// When incrementing the role version, make sure to update the
 	// role version in the asset file used by the UI.
 	// See: web/packages/teleport/src/Roles/templates/role.yaml
-	role, err := NewRoleWithVersion(name, V7, spec)
+	role, err := NewRoleWithVersion(name, V8, spec)
```

Comment:
usually we do not bump the role automatically to allow clusters to be downgraded to major version -1

Reply:
as discussed offline, this only impacts newly created role and doesn't impact downgrades